samples/drivers/can/counter/boards/alif_e3_dk_rtss_he.overlay

```diff
@@ -0,0 +1,18 @@
+/* Copyright (C) 2025 Alif Semiconductor - All Rights Reserved.
```

Comment:
Please switch to Apache 2.0 as this is Zephyr code. Alif's own license should be used only on the SDK side.